doc/classes/AnimationNodeTransition.xml

```diff
@@ -22,37 +22,46 @@
 			<return type="String" />
 			<param index="0" name="input" type="int" />
 			<description>
+				Returns name of input at given index. This name shows in editor next to node input.
+				[code]input[/code] must be in between 0 and 30 (inclusive).
```

Comment:
Avoid unnecessary new lines in descriptions. Each new line creates a new paragraph, and this is excessive here.

Also use `[param input]` to refer to method' parameters. Numbers and other literals must be wrapped in `[code]`. Also we should avoid having magic numbers in the documentation. These limits probably have corresponding constant somewhere which you should link to.

This applies to all descriptions here.

Reply:
Can do, thanks. It's my first so I get some details wrong... However what should I do with the maximum value which apparently is not exposed? (see lyuma's comment below)

Reply:
One possibility is leave those mentions entirely, just informing that index is zero-based. It makes no sense to try to get values higher than enabled inputs amount, so it's ok if it's not defined.

Reply:
Well, if no constant is exposed, I guess it's okay for now to keep it at magic numbers.

Reply:
I left the magic number in `enabled_inputs`, so it's documented somewhere. For the rest, knowing that it's zero-based is enough.